.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: [3.9, 3.10]
```

Comment:
Core only run the test on 3.9: https://github.com/home-assistant/core/blob/dev/.github/workflows/ci.yaml#L723